### DISCUSSION TITLE
Video widget

### DISCUSSION THREAD

**Discussion by mvheimburg** [flet-dev/flet#464] [Ideas]
I really want to be able to display video from a stream of numpy arrays. Possible?

**Comment by FeodorFitsner** [comment]
It could be made for web and mobile only: https://pub.dev/packages/video_player

Do you need it for a desktop app?

**Reply by mvheimburg** [reply]
Thanks for the amazingly quick reply, and great job on the framework. Looks amazing!

Yeah desktop app was the plan. I could always go web instead... :)

**Reply by mvheimburg** [reply]
How about this package? :)

https://pub.dev/packages/flutter_gstreamer_player
